data/helpers.d/hardware

```diff
@@ -53,20 +55,46 @@ ynh_check_ram () {
 			# Use only the amount of free swap
 			ram=$total_swap
 		fi
+    else
+        echo "Uhoh, you should choose --free or --total when using ynh_get_ram" >&2
```

Comment:
Something more formal maybe ?
In any case, a ynh_print_something would be better.